### ISSUE TITLE
Document how admins can set HEAD for repos that don't have a master branch

### ISSUE THREAD

**Issue by bowlofeggs** [repoSpanner/repoSpanner#92]
I pushed the [bodhi](https://github.com/fedora-infra/bodhi) repo into repospanner. When I tried to clone it, the clone failed:

```
$ git clone --config http.sslcert=~/devel/bodhi/admin.crt --config http.sslkey=~/devel/bodhi/admin.key --config http.sslCAinfo=~/devel/bodhi/ca.crt https://n2.home.repospanner.electronsweatshop.com/repo/bodhi.git delme
Cloning into 'delme'...
remote: Welcome to repoSpanner 0.5+1293143, node n2.home.repospanner.electronsweatshop.com
remote: Building packfile
remote: Packfile built, sending
remote: Packfile sent
Receiving objects: 100% (43437/43437), 116.51 MiB | 18.86 MiB/s, done.
warning: remote HEAD refers to nonexistent ref, unable to checkout.
```

This happens because Bodhi doesn't have a master branch. As @puiterwijk mentioned below, there is a command that admins can run to set HEAD to a different ref. We should document that command.

**Comment by puiterwijk** [comment]
So, as we discussed, this is not a bug in repoSpanner, but a consequence of the fact that there's no `master` branch for Bodhi.
repoSpanner creates the `HEAD` ref by default to point to `refs/heads/master`, because figuring out what branch should be default is not really feasible.
Instead, repoSpanner sends a warning to the client, which the admin can fix with `repospanner admin repo edit bodhi HEAD=refs/heads/develop`.

**Comment by bowlofeggs** [comment]
Yeah I see that now - I think I'll turn this ticket into a doc issue and just document this instead. It will affect few projects since it's uncommon not to have a master branch.

**Comment by quanah** [comment]
I have no issue with git fetch when adding fedora's repo as a remote.
